operators/api/v1alpha2/instance_types.go

```diff
@@ -47,6 +49,18 @@ const (
 	EnvironmentPhaseCreationLoopBackoff EnvironmentPhase = "CreationLoopBackoff"
 )
 
+// InstanceCustomizationUrls specifies optional urls for advanced integration features.
+type InstanceCustomizationUrls struct {
+	// URL from which GET the archive to be extracted into Template.ContainerStartupOptions.ContentPath. This field, if set OVERRIDES Template.ContainerStartupOptions.SourceArchiveURL.
+	ContentOrigin string `json:"contentOrigin,omitempty"`
+
+	// URL to which POST an archive with the contents found (at instance termination) in Template.ContainerStartupOptions.ContentPath.
+	ContentDestination string `json:"contentDestination,omitempty"`
+
+	// URL which is periodically checked (with a GET request) to determine automatic instance shutdown. Should return any 2xx status code if the instance has to keep running, any 4xx otherwise.
```

Comment:
Does it make sense to specify it shall return the expected end date/time in a standard format?